src/Test.sol

```diff
@@ -330,6 +330,22 @@ abstract contract Test is DSTest, Script {
         }
     }
 
+    function assertUint128Eq(uint128 a, uint128 b) internal {
+        assertEq(uint256(a), uint256(b));
+    }
+
+    function assertUint64Eq(uint64 a, uint64 b) internal {
+        assertEq(uint256(a), uint256(b));
+    }
+
+    function assertUint96Eq(uint96 a, uint96 b) internal {
+        assertEq(uint256(a), uint256(b));
+    }
+
+    function assertUint32Eq(uint32 a, uint32 b) internal {
+        assertEq(uint256(a), uint256(b));
+    }
+
```

Comment:
```suggestion
    function assertEqUint(uint256 a, uint256 b) internal {
        assertEq(a, b);
    }
```

Reply:
Not sure what you suggest @ZeroEkkusu

Reply:
Sorry, @odyslam, accidentally clicked `Add single comment`. I'll write up a review real quick.